rules/TypeDeclaration/Rector/Param/ParamTypeFromStrictTypedPropertyRector.php

```diff
@@ -111,7 +115,6 @@ private function decorateParamWithType(ClassMethod $classMethod, Param $param):
 
         $paramName = $this->getName($param);
 
-        /** @var Assign[] $assigns */
         $assigns = $this->betterNodeFinder->findInstanceOf((array) $classMethod->getStmts(), Assign::class);
```

Comment:
This is needed for PHPStorm IDE autocomplete.

Reply:
good catch. fixed.